tests/sanity/ignore-2.13.txt

```diff
@@ -1,6 +1,7 @@
 .azure-pipelines/scripts/publish-codecov.py replace-urlopen
 plugins/lookup/etcd.py validate-modules:invalid-documentation
 plugins/lookup/etcd3.py validate-modules:invalid-documentation
+plugins/module_utils/gitlab.py validate-modules:collection-deprecated-version
```

Comment:
Please remove these entries :)

Reply:
(This would have to be a `pylint:collection-deprecated-version` ignore anyway, since it's failing the `pylint` test and not the `validate-modules` test :) )